membership-attribute-service/app/monitoring/SentryLogging.scala

```diff
@@ -1,43 +1,25 @@
 package monitoring
 
-import ch.qos.logback.classic.filter.ThresholdFilter
-import ch.qos.logback.classic.{Logger, LoggerContext}
-import com.getsentry.raven.RavenFactory
-import com.getsentry.raven.logback.SentryAppender
+import io.sentry.Sentry
 import configuration.Config
-import org.slf4j.Logger.ROOT_LOGGER_NAME
-import org.slf4j.LoggerFactory
+import scala.collection.JavaConversions._
+import scala.util.{Failure, Success, Try}
 
 object SentryLogging {
 
   val UserIdentityId = "userIdentityId"
   val UserGoogleId = "userGoogleId"
-  val AllMDCTags = Seq(UserIdentityId, UserGoogleId)
-
-  val RavenOpt = Config.sentryDsn.map(RavenFactory.ravenInstance).toOption
+  val AllMDCTags = Set(UserIdentityId, UserGoogleId)
 
   def init() {
-    RavenOpt match {
-      case None =>
-        play.api.Logger.warn("No Sentry logging configured (OK for dev)")
-        
-      case Some(raven) =>
-        play.api.Logger.info(s"Initialising Sentry logging")
-        val buildInfo: Map[String, Any] = app.BuildInfo.toMap
+    play.api.Logger.info(s"Initialising Sentry logging")
+    Try(Sentry.init(Config.sentryDsn)) match {
```

Comment:
I'd prefer to express the fact that the sentryDsn is optional at the Config level, but I guess that's just personal preference...

Reply:
you mean that it should try to load sentry if config.sentryDsn is set and let it crash if it is set to something invalid?

Reply:
No, sorry, I meant that I'd prefer to make it explicit that "sentry.dsn" might not be defined in the config [here](https://github.com/guardian/members-data-api/pull/304/files#diff-0494131766aaa09e284e48e59b700e11R27) (I don't think it is in DEV/CODE for a number of apps). Feel free to ignore the comment if you think I'm missing the point...